### PR TITLE
Add tools parameters for ssdt17

### PR DESCRIPTION
**Pull request by kwlin**
And now for ssdt17 :) 

Please review/test if this pull request is working. The arguments is a bit different then ssdt15.

### CODE REVIEW

File: automatic/ssdt17/ssdt17.nuspec [hunk 1]
```diff
@@ -12,7 +12,21 @@
...
+The following package parameters can be set:
+
+ * `/analysis[:VSInstanceIds]` - Install SSAS to specified VS2017 instances.
+ * `/integration[:VSInstanceIds]` - Install SSIS to specified VS2017 instances.
+ * `/reporting[:VSInstanceIds]` - Install SSRS to specified VS2017 instances.
+
+VSInstanceIds is the list of target VS 2017 instance ids seperated by comma. It can be absent when there is only one VS 2017 
+instance installed on current machine
+
+To pass parameters, use --params "''" (e.g. choco install ssdt17 [other options] --params="'/analysis /reporting:'").
```
Comment (mikecole) [inline automatic/ssdt17/ssdt17.nuspec:26]: Does the example have an extra colon after `/reporting`?

**Comment by mikecole** [comment]
This one is giving me general MSI installation errors, and I can't find any logs telling me why. :-(

I'm going to play around with it a little. It works with no add-ons, but when including an add-on it fails. I have a wild idea to try next, but as of right now this doesn't appear to be as easy as I thought.

File: automatic/ssdt17/ssdt17.nuspec (hunk 1 above)
Comment (kwlin) [reply automatic/ssdt17/ssdt17.nuspec:26]: Typo. As an example it should have some VS2017 instance names, like /reporting:VsInstance1,VsInstance2

Thanks for the review!

**Comment by kwlin** [comment]
Hmz, I'll try to fix it asap :) maybe my assumption to get parameters from the commandline is wrong. https://chocolatey.org/docs/how-to-parse-package-parameters-argument

That's the only difference between ssdt15 & ssdt17

**Comment by mikecole** [comment]
That would be great. Let me know what you find.

**Pull request closed without merging**